getfundwave/fw-components
# Test for the Fw-web-components: fw-avatar and fw-avatar-group.

Added some test cases for fw-avatar and fw-avatar-group. Still some test cases are to be added. 
Please provide feedback for any improvement.

1. Added required libraries (web-test-runner , open-wc/testing , polymer elements).
2. Test folder contain fw-avatar.test.js and fw-avatar-group.test.js

**fw-avatar-test:** 
- check if the custom element generated is an instance for FwAvatar class
- check if the properties(title, type, color, name, image src) passed to the element  

- [x] test for luminance property is to be added

**fw-avatar-group-test**
- check if the custom element generated is an instance for FwAvatarGroup class
- check for the item property (with and without secondary attribute)
- check for maxCount and header property 
- check if the searchBar and Menu is visible on click event
- check for the primary and secondary actions passed in the property , check if the event associated with them is dispatched on click 

- [x] test to check the emptyStateMsg property is to be added
